fft/launch/fft_qtplot.launch

```diff
@@ -0,0 +1,9 @@
+<launch>
+  <node name="fft_qtplot"
+        pkg="fft" type="fft_qtplot.py"
+        if="$(arg qtplot)">
```

Comment:
Could you add qtplot arg at the beginning?
`<arg name="qtplot" default="false" />`
or
you can merge this codes with `fft_plot.launch` with some additional args.